server/src/disposition/disposition.controller.ts

```diff
@@ -51,7 +51,7 @@ const { deserialize } = new Deserializer({
 @Controller('dispositions')
 export class DispositionController {
   constructor(
-    private readonly dynamicsWebApi:OdataService,
+    private readonly dynamicsWebApi:CrmService,
```

Comment:
Nit: Can we change this to crmService too? I think the consistency will help w readability, and may help with future find and replace

Reply:
Yess good eye, thanks